sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -1480,6 +1483,26 @@ public boolean sendPacketToRegisteredApp(SdlPacket packet) {
 	    				}
 	    			}
 
+					if(packet.getFrameType() == FrameType.Single && packet.getServiceType() == SdlPacket.SERVICE_TYPE_RPC){
```

Comment:
We should first check the size of `cleanedSessionMap` to see if any entries exist. This will prevent us from having to parse every RPC if there is no need to.